### ISSUE TITLE
在运行EasyEdit/multimodal_edit.py中train_MEND_Blip2OPT_VQA()时，始终显示“NAN in logits!!!”

### ISSUE THREAD

**Issue by QinWeijia111** [zjunlp/EasyEdit#526]
问题如上，以下是报错的截图，采用单卡GPU运行

![Image](https://github.com/user-attachments/assets/6a7e01c6-2961-4037-96c6-cf0f22339250)

以下是我的config配置文件（只修改了name和tokenizer_name部分）：
# Model
device: 0

name: Salesforce/blip2-opt-2.7b
model_name: blip2
model_class: Blip2OPT
tokenizer_class: GPT2Tokenizer
tokenizer_name: Salesforce/blip2-opt-2.7b
inner_params:
  - opt_model.model.decoder.layers.29.fc1.weight
  - opt_model.model.decoder.layers.29.fc2.weight
  - opt_model.model.decoder.layers.30.fc1.weight
  - opt_model.model.decoder.layers.30.fc2.weight
  - opt_model.model.decoder.layers.31.fc1.weight
  - opt_model.model.decoder.layers.31.fc2.weight

# Method
alg: MEND
lr: 1e-6
edit_lr: 1e-4
lr_lr: 1e-4
seed: 42
cedit: 0.1
iedit: 0.1
cloc: 1.0
cbase: 1.0
dropout: 0.0
train_base: False
no_grad_layers: null
one_sided: False
n_hidden: 1
hidden_dim: null
init: id
norm: True
combine: True
x_only: False
delta_only: False
act: relu
rank: 1920
mlp_class: IDMLP
shared: True
archive: null

# Train
batch_size: 1
model_save_pt: 5000
silent: False
#max_epochs: 1
max_iters: 30000
log_interval: 200
eval_log_interval: 5000
final_eval: True
val_interval: 5000
early_stop_patience: 20000
early_stop_key: "loss/total_edit_val"
eval_only: False
half: False
debug: False
save: False
verbose: True

val_batch_size: 1
accumulate_bs: 2
val_steps: 500 # only for debug
opt: Adam
grad_clip: 100.

# Output
results_dir: ./results

# Multimodal
exact_match: True
qformer_checkpoint: /root/shared-nvme/dataAndModel/blip2_pretrained_opt2.7b.pth
qformer_name_or_path: bert-base-uncased
state_dict_file: /root/shared-nvme/dataAndModel/eva_vit_g.pth
# image
coco_image: /root/shared-nvme/dataAndModel/
rephrase_image: /root/shared-nvme/dataAndModel/

请求作者大大给一些复现的建议呜呜呜

**Comment by QinWeijia111** [comment]
好像找出来原因了，是因为我在下载模型的时候下载的是Salesforce/blip2-opt-2.7b这个模型，而不是语言模型，这部分搞错了，目前已经没有了以上的问题：

![Image](https://github.com/user-attachments/assets/c419cd77-8c87-42a8-8ace-a175b74e33fd)
